apps/homework/templatetags/homework_tags.py

```diff
@@ -48,12 +48,24 @@ def get_last_grade_teacher(submission):
 def get_item(dictionary, key):
     return dictionary.get(key)
 
+
+def has_item(dictionary, key):
+    return key in dictionary
+
+
+def polish_log_name(log_name):
```

Comment:
This is microscopically minor, but I couldn't help but read this as `Polish log name`. Maybe `format_log_name` would be a better function name. It's all good if this is your preference.

Polish Log
![image](https://user-images.githubusercontent.com/21694127/69993409-9b6cbf00-1500-11ea-96c6-539713fbef5a.png)

Reply:
Lmao, I renamed it to something less delicious.

👍